doc/rados/operations/pools.rst

```diff
@@ -913,6 +913,10 @@ Ceph will list pools and highlight the ``replicated size`` attribute.  By
 default, Ceph creates two replicas of an object (a total of three copies, for a
 size of ``3``).
 
+Managing pools that are flagged ``bulk``
```

Comment:
Above has "with", this is without "with".  Consistent?

Reply:
That's a good question. Since this has raised an issue in the mind of the first-time reader, I'll add the "with".

Reply:
Either way is coolio with me, just so the two instances are consistent.   I might have chosen a flag name other than "bulk" and used "as" vs "with" but w/e